compiler/luci/service/src/Nodes/CircleReshape.test.cpp

```diff
@@ -197,3 +197,79 @@ TEST(ShapeRuleTest, reshape_by_newShape)
   ASSERT_EQ(2, output_shape.dim(0).value());
   ASSERT_EQ(12, output_shape.dim(1).value());
 }
+
+TEST(ShapeRuleTest, reshape_by_newShape_dynamic)
```

Comment:
plz add negative tests so that `count(neg) >= count(pos)`

Reply:
I see, I've added 2 negative tests to neutralize the influence of the new ones ;)